OverlayPlugin.Core/EventSources/FFXIVRequiredEventSource.cs

```diff
@@ -242,8 +231,13 @@ struct PartyMember
             // Raw job id.
             public int job;
             public int level;
-            // In immediate party (true), vs in alliance (false).
+            // @deprecated, please use partyType
             public bool inParty;
+            public long contentID;
+            public byte flags;
+            public uint objectID;
+            public ushort territoryType;
```

Comment:
Are all of these worth sending? territoryType and contentID are in many other log lines.  I don't know what ObjectID and flags are (or why they're worth having).

Reply:
`objectID` is the player ID, e.g. `10001234`. That's sent currently as a string for some reason, I didn't see any harm in including the actual numerical version.

`territoryType` isn't included in other log lines if the player is in another instance/zone, since they'll never generate a log line from another zone.

`flags` is potentially useful once we can decode it.

Reply:
I thought we didn't get info about party members in cross-world parties (and that this behaved the same in terms of party events as before this PR).  If that's true, then how does this work?

Reply:
This would be for same-world parties. e.g. if I'm in Limsa and someone else in my party is off in Ultima Thule, that should still contain the `territoryType` value because it's not cross-world.

Reply:
Ah, fair.  I'm still not sure I understand how this would be useful, but that makes sense.